mkdocs.yml

```diff
@@ -1,6 +1,6 @@
 site_name: django-user-management
 pages:
-- [index.md, Home]
+- [README.md, Home]
```

Comment:
Oh ok I see what you want to do. In that case you probably need to rename the file too.